### ISSUE TITLE
coverImage not working as a splash image or playlist icon

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/flowplayer-core#484]
```
What steps will reproduce the problem?
1. Create XML (or in-stream) Playlist of two or more video/audio files (set to 
autoPlay: false)
2. Assign a coverImage clip property to use as splash image
3. See the empty / black player, the coverImage is never displayed (unless you 
actually create a separate playlist item?)

What is the expected output? What do you see instead?

I would expect to be able to use coverImage as a splash image (and subsequently 
Playlist icons).
I see an empty player, since I am not able to use <a><img/></a> format with our 
dynamic XML playlists.


What version of the product are you using? On what operating system?

Flowplayer Commercial v3.2.7
OS X 10.7.3 / Firefox 11


Please provide any additional information below.

I've never really understood the logic around Flowplayer's solution to serving 
up a splash image for a given playlist entry. It almost seems like an 
afterthought? In fact, the current working methods I've seen are more of a 
hack, requiring you to increase your playlist size (offsetting original 
indexes, etc) by representing each splash image as a playlist entry (sometimes 
defining duration:0 ), and subsequent autoPlay: false for the actual intended 
video, etc.

Why can't coverImage on a given clip simply show the image? (when not set to 
autoPlay, of course) 

Don't get me wrong, I absolutely love Flowplayer -- the advanced API, and 
granular customization options. It's very powerful. JWPlayer (from 
longtailvideo.com), on the other hand, has done things exactly right when it 
comes to splash images / playlist icons, IMHO. They just work, and exactly as 
you'd expect.

The issue I'm facing is that we're using fully dynamic MRSS XML playlists. Each 
playlist entry represents a multimedia asset (video or audio), with the 
corresponding screenshot stored at the clip-level (e.g. fp:clip coverImage). 
Unfortunately, this coverImage only seems to apply to audio files, such as with 
mp3, but DURING playback. 

Does anyone have a patch or other solution that can circumvent having to 
manipulate the playlist entries in order to get a simple splash image to work 
directly from XML? (or inline playlist, for that matter)

I realize the other option is to use a nested IMG tag in the flowplayer HTML 
container, but that model is not well supported for dynamic XML playlists 
containing all the meta data, etc.

Further, I can't seem to find any built-in playlist menu with Flowplayer. I've 
used the HTML/JS plugin in the past, but that was using in-line playlists. Does 
anyone know if this supports XML playlists? Has anyone had success getting 
playlist icons to work, such as this: 

http://flowplayer.org/forum/4/68241

In a nutshell, is Flowplayer in any way capable of showing a coverImage for 
playlist items <em>without</em> having to be a playlist entry itself? If not, 
are there any plans to possibly support this?
```

Original issue reported on code.google.com by `pixelchu...@gmail.com` on 6 Mar 2012 at 8:49
